huntsman/tests/test_huntsman.py

```diff
@@ -252,8 +252,6 @@ def test_entering_darks_state(pocs, db):
 
     pocs.state = 'taking_darks'
```

Comment:
This is not testing whether it can enter the `taking_darks` state (I don't actually think it can at the moment because there is no mention of `taking_darks` in the state table, see comment below).

So first you assert that the conditions are right to enter the dark state. So you need it to be a) dark, b) weather not safe and c) in the correct state to transition into `taking_darks`. It looks like this should be `parked` from the rest of your code, not `sleeping`. You will then need something like:

```
        assert pocs.next_state == "taking_darks"
        pocs.goto_next_state()
        assert pocs.state == "taking_darks"
``` 

for this to actually test whether it successfully enters the state. Finally, I think this test would be better suited to being in `tests/test_states.py` rather than `tests/test_huntsman.py`. I would suggest taking a look at some of the other tests in `tests/test_states.py` if this is not clear.